### DISCUSSION TITLE
Um projeto mal pensando e cheio de problemas pode afetar sua saúde?

### DISCUSSION THREAD

**Discussion by daniellucindo** [frontendbr/forum#453] [Discussão]

**Comment by woliveiras** [comment]
Rapaz, acho que o projeto em si não faz mal a ninguém. É só um monte de código mal feito que vamos arrumando com o tempo (ou não).

O que faz mal é uma **má gestão**.

Cobranças indevidas, prazos muito loucos, mandam fazer uma coisa depois mandam desfazer, te tiram a liberdade de tomar decisões, dentre outras maneiras que uma gestão ruim utiliza pra pressionar a rápida entrega de novas features.

Isso faz mal tanto para o desenvolvedor, que fica num pico de stress bem alto, quanto para o projeto, que ao invés de ser arrumado começa a ter mais e mais código ruim pra contornar problemas e não para resolve-los pra que a entrega seja no tempo exigido pela gestão.

**Comment by felipefialho** [comment]
Trabalhamos com a mente, então, naturalmente o cerebro buga quando por exemplo, uma funcionalidade muda várias vezes por má definição no inicio do projeto, o resultado disso é alguns problemas psicológicos chatos.

Mas apesar de dificil, não acho que você deve estender problemas de projetos e profissionais para outras areas da sua vida. 

Tente desligar no fim de semana, pegar férias, enfim.. ter uma vida normal, que esses problemas tendem a diminuir.

**Comment by ericvvf** [comment]
Aproveitando o que disse o @woliveiras, acredito que, na maioria das vezes, o caos é criado pela má gestão. Acredito que o stress em si sempre irá existir, e é importante e essencial saber administrá-lo bem. O pior caso pelo qual passei, foi trabalhar em uma empresa em que os projetos eram geridos por pessoas que não sabiam nada de desenvolvimento, e as cobranças indevidas, prazos passados, não faziam menor sentido para nossa equipe. E sim, por imaturidade minha, isso foi bem nocivo, em alguns momentos, à minha saúde.

Como disse o @LFeh, nestes casos, separar as coisas, ou em outras palavras, não levar o problema pra casa, é essencial.

**Comment by paulinhapenedo** [comment]
E quando a demanda/prazo obriga que você leve o projeto para casa? Seja para estudar uma tecnologia, testar metodologias ou algo do tipo? Pergunto isso porque isso acontece comigo e também não sei o que fazer.

**Comment by dionarodrigues** [comment]
O melhor ambiente de trabalho é aquele nos faz se sentir bem. Claro que há muitas variáveis envolvidas e, assim como em nossa vida pessoal, há momentos que estamos mais envolvidos e mais relaxados, enquanto em outros não. O importante é analisar o todo.

Se um sentimento de opressão persiste por um tempo considerável, talvez algo esteja errado. 

Procure fazer o que o @LFeh disse: utilize seus finais de semana para relaxar. Pois, às vezes você também pode estar se cobrando demais.

**Comment by kvnol** [comment]
Já fiquei com muita dor de cabeça por causa de trampo e minha dica é **"não leve serviço pra casa"** (com o perdão do trocadilho pra galera do home office), após sair de um dia cheio de trabalho, com prazo mal definido, cliente no ouvido: tente esquecer seu trabalho, ouça uma música, procure um hobby (o meu é a guitarra) e relaxe.

**Comment by felipefialho** [comment]
@paulinhapenedo A sua questão é muito boa.

Trabalhamos em uma área em que a tecnologia utilizada evolui absurdamente rápida. E um desenvolvedor que deseja alcançar um bom nível, PRECISA estudar muito, não tem como fugir.

Então é importante se organizar para encaixar essas tarefas de estudo no seu dia-a-dia sem afetar sua vida pessoal. Porque isso vai acontecer a vida toda. Encare isso como uma faculdade.

Mas é um ponto diferente do problema do @daniellucindo, percebeu? Ele está lidando com um projeto complicado e isso está afetando a vida dele. No seu caso, em tese, é estudo por si só.

Em geral, tento encaixar três vezes por semana. A única regra que coloco, é evitar ao máximo abrir códigos no fim de semana.
